airflow/utils/log/LoggingMixin.py

```diff
@@ -27,18 +27,18 @@ class LoggingMixin(object):
     """
 
     @property
-    def logger(self):
+    def log(self):
```

Comment:
Do we need to worry about this being non-backwards compatible from 1.8 to 1.9?  (i.e. I have a custom operator which calls `self.logger.info`?

(I'm not sure what version numbering scheme Airflow follows. I guess it's not semver though, right?)

Reply:
Good catch. We can't break backwards compatibility until 2.0, `self.logger` needs to exist/work and could warn if it's planning for deprecation. If it warns it should have a counter and warn a certain number of times only before silencing itself.

Reply:
Hi @mistercrunch @ashb,

Thanks for pointing out. I've added the `.logger` with a deprecation warning. I did not implement the counter as you suggested as this is already implemented by Python's warning framework.
https://docs.python.org/2/library/warnings.html

This is explained under the warnings filter section. By default Python will only print each warning only once. You can set it more verbose if you want to track down all the warnings in your application. This is already handled by Python, so we don't have to integrate it into Airflow itself.